src/components/HdModal.vue

```diff
@@ -6,7 +6,7 @@
       :class="{ 'hd-modal--with-icon': withIcon }"
     >
       <div class="hd-modal__overlay" />
-      <div class="hd-modal__container">
+      <div class="hd-modal__container" v-if="!isCleanSheet">
```

Comment:
Shouldn't we put `v-if` right before the `class` attribute?

Reply:
Sure. Done.